server/routes/api.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express')
+const router = express.Router()
+const request = require('axios')
+
+const City = require('../models/City')
+
+function apireq (city) {
+  return request(`http://api.openweathermap.org/data/2.5/weather?q=${city}&appid=d98539d8bfaec863c4dbb2f0d8f3dc40&units=metric`)
+}
+
+router.get('/city/:cityName', async function (req, res) {
+  const city = req.params.cityName
+  const name = city.charAt(0).toUpperCase() + city.slice(1)
+  const data = await apireq(name)
+    .catch(function (err) {
+      console.error(err)
+    })
```

Comment:
if you are using async-await, there is no need to use a `then` or `catch` function.
for this, you have try catch, [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch)